plugins/ipqualityscore/plugin.spec.yaml

```
@@ -0,0 +1,795 @@
+        server:
+          title: Server
+          description: The server banner of the domain's IP address For example  nginx Value will be NA if unavailable
+          type: string
+          required: false
+          example: 'nginx/1.16.0'
+        domain_rank:
+          title: Domain Rank
+          description: Estimated popularity rank of website globally Value is 0 if the domain is unranked or has low traffic
+          type: integer
+          required: false
+          example: 0
+        dns_valid:
+          title: DNS Valid
+          description: The domain has valid DNS records?
+          type: boolean
+          required: false
+          example: false
+        parking:
+          title: Parking
+          description: Is the domain currently parked with a for sale notice?
+          type: boolean
+          required: false
+          example: false
+        spamming:
+          title: Spamming
+          description: Is the domain  associated with email SPAM or abusive email addresses?
+          type: boolean
+          required: false
+          example: false
+        malware:
+          title: Malware
+          description: Is this domain associated with malware or viruses?
+          type: boolean
+          required: false
+          example: false
+        phishing:
+          title: Phishing
+          description: Is this domain associated with malicious phishing behavior?
+          type: boolean
+          required: false
+          example: false
+        suspicious:
+          title: Suspicious
+          description: Is this URL suspected of being malicious or used for phishing or abuse? Use in conjunction with the risk_score as a confidence level
+          type: boolean
+          required: false
+          example: false
+        adult:
+          title: Adult
+          description: Is this URL or domain hosting dating or adult content?
+          type: boolean
+          required: false
+          example: false
+        risk_score:
+          title: Risk Score
+          description: The IPQS risk score which estimates the confidence level for malicious URL detection Risk Scores 85+ are high risk, while Risk Scores = 100 are confirmed as accurate
+          type: integer
+          required: false
+          example: 0
+        category:
+          title: Category
+          description: Website classification and category related to the content and industry of the site Over 70 categories are available including Video Streaming, Trackers, Gaming, Privacy, Advertising, Hacking, Malicious, Phishing, etc The value will be NA if unknown
+          type: string
+          required: false
+          example: 'Search Engines'
+        domain_age:
+          title: Domain Age
+          description: A human description of when this domain was registered
+          type: object
+          required: false
+          example: '{human: 26 years ago,timestamp: 874296000,iso: 1997-09-15T00:00:00-04:00}'
```

Can these object examples please be updated to a valid JSON?

```suggestion
          example: '{"human": "26 years ago", "timestamp": "874296000,iso: 1997-09-15T00:00:00-04:00"}'
```